internal/db/migrate.go

```diff
@@ -1,34 +1,18 @@
 package db
 
 import (
-	"embed"
-	_ "embed"
-	"fmt"
 	"sublinks/sublinks-federation/internal/log"
+	"sublinks/sublinks-federation/internal/model"
 
-	"github.com/golang-migrate/migrate/v4"
-	"github.com/golang-migrate/migrate/v4/database/mysql"
 	_ "github.com/golang-migrate/migrate/v4/source/file"
-	"github.com/golang-migrate/migrate/v4/source/iofs"
 )
 
-//go:embed "migrations"
-var migrations embed.FS
-
 func (d *PostgresDB) RunMigrations() {
 	logger := log.NewLogger("db migrations")
 	logger.Debug("Running migrations...")
-	driver, err := mysql.WithInstance(d.DB, &mysql.Config{})
-	if err != nil {
-		logger.Fatal("Error getting MySQL driver", err)
-	}
-	source, _ := iofs.New(migrations, "migrations")
-	m, err := migrate.NewWithInstance("iofs", source, "mysql", driver)
+	err := d.DB.AutoMigrate(&model.Actor{}, &model.Post{})
```

Comment:
I feel like there should be a better way to do this

Reply:
I was using Goose here earlier. My comment was more that I didn't like that I had to manually update the list of objects. There should be a way to have the list built dynamically instead (or let models self register somehow)